common/config_utils.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Provides helper functions to obtain configurations."""
+
+
+def validate_and_expand(config):
+    """Validates |config| and returns the enlarged one."""
```

Comment:
Right. Maybe this will not happen in the new arch. In the old arch, some arguments will be assigned with a default one and added to the dictionary. Like this:

https://github.com/google/fuzzbench/blob/140e69107d725611290a05d6369f159337d61bdf/experiment/run_experiment.py#L79

https://github.com/google/fuzzbench/blob/140e69107d725611290a05d6369f159337d61bdf/experiment/run_experiment.py#L135-L136

Also,

https://github.com/google/fuzzbench/blob/140e69107d725611290a05d6369f159337d61bdf/experiment/run_experiment.py#L214